BiometricKit/BiometricKitIdentity.h

```diff
@@ -1,10 +1,19 @@
+API_AVAILABLE(ios(7.0))
 @interface BiometricKitIdentity : NSObject <NSSecureCoding, NSCopying>
 
-@property (nonatomic, retain) NSUUID * uuid;
-@property (nonatomic, assign) NSUInteger userID;
-@property (nonatomic, assign) NSInteger type;
-@property (nonatomic, assign) NSInteger attribute;
-@property (nonatomic, assign) NSInteger entity;
-@property (nonatomic, copy) NSString * name;
++ (instancetype)biometricKitIdentity;
+
+@property (nonatomic, retain) NSUUID *uuid;
+
+@property (nonatomic, assign) uid_t userID API_AVAILABLE(ios(10.0));
+@property (nonatomic, assign) int type;
+@property (nonatomic, assign) int attribute;
+@property (nonatomic, assign) int entity;
```

Comment:
Are these a regression?

Reply:
I don't believe so. The properties were previously listed as `NSInteger`s however, these headers were dumped from a 64-bit environment, where an `NSInteger` should be at least `long`. I believe these values are enums (without an underlying type specified- defaulting to `int`), however I have not been able to glean much insight into the values of the enum.

Reply:
I just saw [this comment](https://github.com/theos/headers/pull/23#issue-331809461), which reinforces these type choices